pom.xml

```diff
@@ -376,7 +375,7 @@
                         <exclude>**/src/test/resources/**</exclude>
                         <exclude>**/target/**</exclude>
                     </excludes>
-                    <lineEnding>LF</lineEnding>
+                    <lineEnding>AUTO</lineEnding>
```

Comment:
LF is unix style.  Not windows.  I realize you use a mac.  Not everyone does.  AUTO is box dependent.  Sure you can have your AUTO set to something else but that is box specific.  AUTO is on git attributes.  It has been since the original rolled into git.  That in fact is your AUTO.  It's no different.  I had nothing but issues locally with these settings.  I couldn't do anything.  I had been making changes and pushing them all up so they would build in linux.  That was the only way for me to verify a thing.  I don't have a mac and I'm not running a docker instance with linux that often.  So I relied on the externals.  I work on some 70+ projects, this is the first outside of MULE ESB that became completely unstable after merging all this code in to the point where I cannot build any longer.  If 70+ projects have no issues, then it sort of indicates a problem here.  Please take the time to look at the actual builds.  You will see that it didn't format anything.  It skipped because this is correct.  App voyer isn't doing it correctly but I fully expected that given what was going on before.  It's checking out wrong most likey as-is meaning LF and thus it will cause formatting there.  But travis is right and it's ok.